alfresco-transform-tika/alfresco-transform-tika/src/main/java/org/alfresco/transformer/metadataExtractors/IPTCMetadataExtractor.java

```diff
@@ -65,9 +63,29 @@ protected Parser getParser() {
      */
     @Override
     protected Map<String, Serializable> extractSpecific(Metadata metadata, Map<String, Serializable> properties,
-            Map<String, String> headers) {
-
+            Map<String, String> headers) 
+    {
         properties = new TikaAutoMetadataExtractor().extractSpecific(metadata, properties, headers);
+        if (parser.getSeparator()!=null)

```

Comment:
ExifToolParser can have a null value for `separator` / `getSeparator()`.
This can happen if ExifTool is not correctly installed on the server hosting the T-engine or there is an error in the config used to initialise the parser. In this scenario. the T-engine will continue to work, but it will not perform IPTC metadata extraction, only alfresco:cm and exif.

Reply:
OK. Is there a warning if exiftool not installed correctly (eg. via probe/testcall on startup) ?

Reply:
Not by default. The config executes `exiftool -ver` to see if  it's installed. If it doesn't get the number back it doesn't instantiate a parser. 
If there is no parser created I have added an error log that suggested ExifTool is not installed correctly.

Reply:
Thanks